x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_expression.tsx

```diff
@@ -113,8 +113,8 @@ export function MetricChart({
         flexDirection: 'column',
         justifyContent: 'center',
         alignItems: 'center',
-        maxWidth: '100%',
-        maxHeight: '100%',
+        width: '100%',
```

Comment:
All these inline styles should also be ported to SASS and just apply a className

Reply:
Yeah. The only reason they're inlined was because this plugin has no sass file, and it didn't seem like a big win to create one just for this, given that we aren't going to override them in any meaningful way or use the cascade. But I'm OK with doing that, if you want.